project/settings/development.py

```
@@ -35,3 +35,10 @@
     }
 }
 
+# Email server configuration
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_HOST_USER = config("EMAIL_HOST_USER")
```

Ruff (0.8.2)

`config` may be undefined, or defined from star imports

(F405)